### ISSUE TITLE
[v1.0-Scenarios-CalendarRemainder.json]: Label 'Snooze for' is not associated with '5 minutes' button in 'CalendarRemainder.json' card when user navigate to '5 minutes' button in backward navigation.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7575]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

This issue can be originally tracked using below URL as it is already logged under AC-ADO.

https://microsoft.visualstudio.com/OS/_workitems/edit/37842122

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
this doesn't rerpo in the latest sample app.

Please try the app version that is greater than equal to release 141.
